airflow/providers/google/cloud/hooks/workflows.py

```diff
@@ -0,0 +1,401 @@
+        """
+        metadata = metadata or ()
+        client = self.get_workflows_client()
+        name = f"projects/{project_id}/locations/{location}/workflows/{workflow_id}"
+        return client.delete_workflow(request={"name": name}, retry=retry, timeout=timeout, metadata=metadata)
+
+    @GoogleBaseHook.fallback_to_default_project_id
+    def list_workflows(
+        self,
+        location: str,
+        project_id: str,
+        filter_: Optional[str] = None,
+        order_by: Optional[str] = None,
+        retry: Optional[Retry] = None,
+        timeout: Optional[float] = None,
+        metadata: Optional[Sequence[Tuple[str, str]]] = None,
+    ) -> ListWorkflowsPager:
+        """
+        Lists Workflows in a given project and location.
+        The default order is not specified.
+
+        :param filter_: Filter to restrict results to specific workflows.
+        :type filter_: str
+        :param order_by: Comma-separated list of fields that that
+            specify the order of the results. Default sorting order for a field is ascending.
+            To specify descending order for a field, append a "desc" suffix.
+            If not specified, the results will be returned in an unspecified order.
+        :type order_by: str
+        :param project_id: Required. The ID of the Google Cloud project the cluster belongs to.
+        :type project_id: str
+        :param location: Required. The GCP region in which to handle the request.
+        :type location: str
+        :param retry: A retry object used to retry requests. If ``None`` is specified, requests will not be
+            retried.
+        :type retry: google.api_core.retry.Retry
+        :param timeout: The amount of time, in seconds, to wait for the request to complete. Note that if
+            ``retry`` is specified, the timeout applies to each individual attempt.
+        :type timeout: float
+        :param metadata: Additional metadata that is provided to the method.
+        :type metadata: Sequence[Tuple[str, str]]
+        """
+        metadata = metadata or ()
+        client = self.get_workflows_client()
+        parent = f"projects/{project_id}/locations/{location}"
+
+        return client.list_workflows(
+            request={"parent": parent, "filter": filter_, "order_by": order_by},
+            retry=retry,
+            timeout=timeout,
+            metadata=metadata,
+        )
+
+    @GoogleBaseHook.fallback_to_default_project_id
+    def create_execution(
+        self,
+        workflow_id: str,
+        location: str,
+        project_id: str,
+        execution: Dict,
+        retry: Optional[Retry] = None,
+        timeout: Optional[float] = None,
+        metadata: Optional[Sequence[Tuple[str, str]]] = None,
+    ) -> Execution:
+        """
+        Creates a new execution using the latest revision of
+        the given workflow.
+
+        :param execution: Required. Execution to be created.
```

Comment:
WDYT about extending information about execution?
```suggestion
        :param execution: Required. Input parameters of the execution represented as a dictionary.
```

https://cloud.google.com/workflows/docs/reference/executions/rest/v1/projects.locations.workflows.executions#Execution